subsys/bluetooth/controller/ll_sw/ull_iso_types.h

```diff
@@ -9,11 +9,18 @@
 #define LL_BIS_ADV_HANDLE_BASE BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE
 #define LL_BIS_ADV_IDX_FROM_HANDLE(conn_handle) \
 	((conn_handle) - (LL_BIS_ADV_HANDLE_BASE))
+/* Conditional compile to prevent coverity issue CWE570, comparison of unsigned int to 0 */
+#if defined(LL_BIS_ADV_HANDLE_BASE)
```

Comment:
If `LL_BIS_ADV_HANDLE_BASE` is defined, but is defined as 0, is `#if defined(LL_BIS_ADV_HANDLE_BASE)` still applied? 

I am wondering if this should have been `#if LL_BIS_ADV_HANDLE_BASE > 0` instead